packages/backend-core/src/events/identification.ts

```diff
@@ -166,7 +167,10 @@ const identifyUser = async (
   const type = IdentityType.USER
   let builder = user.builder?.global || false
   let admin = user.admin?.global || false
-  let providerType = user.providerType
+  let providerType
+  if (isSSOUser(user)) {
+    providerType = user.providerType
```

Comment:
If it's a non sso user, is the provider type undefined?

Reply:
yep that's the intention, it's an optional field on the identity